chapters/en/slides/module1/module1_01.Rmd

```diff
@@ -61,242 +64,122 @@ as we will see later.
-<td>8.15</td>
-<td>8.0</td>
-<td>5.56
-</td></tr>
-<tr>
-<td>7.0</td>
-<td>4.82</td>
-<td>7.0</td>
-<td>7.26</td>
-<td>7.0</td>
-<td>6.42</td>
-<td>8.0</td>
-<td>7.91
-</td></tr>
-<tr>
-<td>5.0</td>
-<td>5.68</td>
-<td>5.0</td>
-<td>4.74</td>
-<td>5.0</td>
-<td>5.73</td>
-<td>8.0</td>
-<td>6.89
-</td></tr></tbody></table>
+```{python echo=FALSE, results='asis'}
+# This is the code that generates the table below,
+# and setting results to `'asis'` ensures it is rendered as HTML
+# instead of being wrapped in an output code block
+import altair as alt
+from vega_datasets import data
+
+anscombe = data.anscombe()#.query('Series != "IV"')
+anscombe['Set'] = anscombe['Series'].map({'I': 'A', 'II': 'B', 'III': 'C', 'IV': 'D'})
+anscombe = anscombe.drop(columns='Series')
+anscombe.loc[anscombe['Y'] == anscombe['Y'].max(), 'Y'] = 8.5
+
+for set_name, df in anscombe.groupby('Set'):
+    df = df.drop(columns='Set')
+    df.columns = pd.MultiIndex.from_tuples([(set_name, 'X'), (set_name, 'Y')])
+    print(
+        df
+        .to_html(index=False)
+        .replace('class="dataframe"', 'style="width:70px; float:left; margin-right:40px"')
+        .replace('<thead>',  '<thead style="text-align: center">'))
+```
 
 Notes:
-Think about it, numbers have been around for about 5,000 years.
-Visual systems, on the other hand, have undergone refinement for over
-500,000,000 years by training with a lethal, evolutionary cost function.
-While we need to train ourselves to recognize structure in numerical
-data, we have evolved to instinctively recognize visual patterns and to
-accurately judge properties such as colors and distances between
-objects. Drawing upon this complex build-in apparatus, we can arrive at
-conclusion much faster by glancing at visual representations of data
+Think about it,
+numbers have only been around for about 5,000 years.
+Visual systems on the other hand,
+have undergone refinement for over 500,000,000 years
+by training with a lethal, evolutionary cost function.
+While we need to train ourselves to recognize structure in numerical data,
+we have evolved to instinctively recognize visual patterns
+and to accurately judge properties such as colors and distances between objects.
+Drawing upon this complex build-in apparatus,
+we can arrive at conclusion much faster by glancing at visual representations of data
 rather than numerical representations.
 
 For example, have a look at the
-four sets of numbers in the table below, [put together by statistician
-Francis Anscombe in the
-70s.](https://en.wikipedia.org/wiki/Anscombe%27s_quartet) Can you see
-how the sets are different from each other by just glancing at the
-table?
+four sets of numbers in the table on the slide.
+This is a slightly modified version of the original,
+[which was put together by statistician Francis Anscombe in the
+70s.](https://en.wikipedia.org/wiki/Anscombe%27s_quartet)
+Can you see how the sets are different from each other
+by just glancing at the table?
```

Comment:
again, I would use the phrasing "Can you see the differences in trends or patterns between these four sets of numbers?"